configs/relengworker-prod/config.yml

```diff
@@ -701,6 +701,34 @@ worker_types:
         capacity_ratio: 1.0
         min_replicas: 0
 
+  - worker_type: mobile-1-tree
+    provisioner: scriptworker-k8s
+    root_url: "https://firefox-ci-tc.services.mozilla.com"
+    deployment_namespace: prod-tree
+    deployment_name: tree-prod-relengworker-firefoxci-mobile-1-1
+    autoscale:
+      algorithm: slo
+      args:
+        max_replicas: 1
+        avg_task_duration: 60
+        slo_seconds: 120
+        capacity_ratio: 1.0
+        min_replicas: 0
+
+  - worker_type: mobile-3-tree
+    provisioner: scriptworker-k8s
+    root_url: "https://firefox-ci-tc.services.mozilla.com"
+    deployment_namespace: prod-tree
+    deployment_name: tree-prod-relengworker-firefoxci-mobile-3-1
+    autoscale:
+      algorithm: slo
+      args:
+        max_replicas: 3
+        avg_task_duration: 300
```

Comment:
I think it's good that this is bumped. Unclear how long the `git clone` operation will take on the scriptworker side so it's good to guard ourselves here 👍

Reply:
I just copied what we had for `gecko-tree-3`. I guess the fenix repo is much simpler than `mozilla-release` so 300 should be good.